.github/workflows/nodejs.yml

```diff
@@ -9,60 +9,15 @@ on:
     branches:
       - '**'
 
-env:
-  CI: true
-
 jobs:
-  lint:
-    name: Lint on Node.js ${{ matrix.node }} and ${{ matrix.os }}
-
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        node: [12]
-        os: [ubuntu-latest]
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Use Node.js ${{ matrix.node }}
-        uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node }}
-
-      - name: Install latest npm
-        run: npm install --global npm@latest
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: Lint
-        run: npm run lint
-
   test:
-    name: Test on Node.js ${{ matrix.node }}
-
     runs-on: ubuntu-latest
-
-    strategy:
-      fail-fast: false
-      matrix:
-        node: [10, 12]
-
     steps:
       - uses: actions/checkout@v2
-
-      - name: Use Node.js ${{ matrix.node }}
-        uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
-          node-version: ${{ matrix.node }}
-
-      - name: Install latest npm
-        run: npm install --global npm@latest
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: Test
-        run: npm test
+          node-version: '10'
```

Comment:
[note] We assume to use Node.js 10 in `package.json`:

https://github.com/stylelint/stylelint-demo/blob/6e04fc0458656a1eeb8a1d96387db6972a7435ff/package.json#L208-L210

Reply:
Let's move everything to 14 to make releasing easier.

Reply:
Thanks for the feedback. I've updated to v12 instead of v14 via ib687bc7.

Because some packages do not work on Node v14. See below:

```console
$ npm ci
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

added 1779 packages, and audited 1780 packages in 16s

190 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

Reply:
It seems we need to update also Webpack... (it's a hard task) 🤔 

```console
$ npm ls fsevents@1
stylelint-demo@0.0.1 /Users/koba/git/stylelint/stylelint-demo
└─┬ webpack@4.46.0
  └─┬ watchpack@1.7.5
    └─┬ watchpack-chokidar2@2.0.1
      └─┬ chokidar@2.1.8
        └── fsevents@1.2.13
```

Reply:
12 SGTM.

> It seems we need to update also Webpack... (it's a hard task) 🤔

Yep, it may not be easy. It may also need to wait until stylelint is using PostCSS@8 because of [conflicts with the postcss-loader](https://github.com/stylelint/stylelint-demo/issues/237).